tests/test_tpt_reports.py

```diff
@@ -112,3 +112,26 @@ def test_bad_log_level():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
         assert return_code == 1, "main() should exit with error"
```

Comment:
In line with my other suggestion, make test error message more explicit:
```suggestion
        assert return_code == 1, "main() should exit with error return code 1"
```